spec/core_functions/color/legacy.hrx

```diff
@@ -0,0 +1,175 @@
+<===> rgb/input.scss
```

Comment:
completely new functions like this won't be implemented in `libsass` so we can note that right at the beginning. 

```suggestion
<===> options.yml
---
:ignore_for:
- libsass

<===>
===================================================================
<===> rgb/input.scss
```